docs/book/src/topics/addons.md

```diff
@@ -149,9 +149,7 @@ kubectl apply -f kube-flannel.yml
 
 # External Cloud Provider
 
-To deploy a cluster using [external cloud provider](https://github.com/kubernetes-sigs/cloud-provider-azure), create a cluster configuration with the [external cloud provider template](https://raw.githubusercontent.com/kubernetes-sigs/cluster-api-provider-azure/main/templates/cluster-template-external-cloud-provider.yaml).
-
-After the cluster has provisioned, install the `cloud-provider-azure` components using the official helm chart:
+To install [external cloud provider](https://github.com/kubernetes-sigs/cloud-provider-azure) components on a self-managed cluster using the official helm chart:
```

Comment:
Should we specify that the external cloud provider is now the default/preferred method as opposed to the in-tree provider?

Reply:
Great suggestion, added